src/Microsoft.Health.Dicom.Functions/Registration/ReplaySafeUpdateMeter.cs

```diff
@@ -0,0 +1,36 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using EnsureThat;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Health.Dicom.Core.Features.Telemetry;
+
+namespace Microsoft.Health.Dicom.Functions.Registration;
+
+public class ReplaySafeUpdateMeter
```

Comment:
I don't think this is the abstraction we want to be "replay-safe" as it's not reusable. Instead, this should probably be a Counter

Reply:
Thanks for pointing it out. I have made changed it to a counter.